### DISCUSSION TITLE
Related Works

### DISCUSSION THREAD

**Discussion by dizys** [nyu-multicore/k-means#2] [Ideas]
Related works and possible comparison candidates are posted here.

**Comment by e0kt** [comment]
## ParaKMeans: Implementation of a parallelized K-means algorithm suitable for general laboratory use
Link: [BMC Bioinformatics](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-9-200)
Document download:
[ParaKMeans - Implementation of a parallelized K-means algorithm suitable for general laboratory use.pdf](https://github.com/nyu-multicore/k-means/files/8457630/ParaKMeans.-.Implementation.of.a.parallelized.K-means.algorithm.suitable.for.general.laboratory.use.pdf)

**Reply by dizys** [reply]
https://www.researchgate.net/publication/226494413_Parallel_kh-Means_Clustering_for_Large_Data_Sets

**Comment by dizys** [comment]
## Parallel k/h-Means Clustering for Large Data Sets
Link: [ResearchGate](https://www.researchgate.net/publication/226494413_Parallel_kh-Means_Clustering_for_Large_Data_Sets)
Document download: [Parallel_processing_of_enhanced_K-means_using_OpenMP.pdf](https://github.com/nyu-multicore/k-means/files/8457627/Parallel_processing_of_enhanced_K-means_using_OpenMP.pdf)

**Comment by dizys** [comment]
## Accelerating K-Means Clustering with Parallel Implementations and GPU computing
Link: [IEEE](https://ieeexplore.ieee.org/document/7322467)
Document Download: [HPEC'15-Janki.pdf](https://github.com/nyu-multicore/k-means/files/8457658/HPEC.15-Janki.pdf)
